src/Fable.React/Fable.Helpers.ReactServer.fs

```diff
@@ -614,20 +611,20 @@ let private renderHtmlAttr (html:StringBuilder) (attr: HTMLAttr) =
   | Unselectable v -> boolAttr html "unselectable" v
   #if !FABLE_COMPILER
   | Style cssList ->
-    html.Append "style" |> ignore
-    html.Append "=\"" |> ignore
+    html.Write "style"
+    html.Write "=\""
 
-    for cssProp in cssList do
-      renderCssProp html cssProp
-    if not(List.isEmpty cssList) then
-      html.Remove (html.Length - 1, 1) |> ignore
-    html.Append '"' |> ignore
+    cssList
+    |> List.iteri( fun i cssProp ->
+        if i > 0 then html.Write(';')
+        renderCssProp html cssProp)
```

Comment:
Instead on removing the lase `;`, we write it before each prop except the first one.
TextWriter doesn't provide `Remove` and not writing in a first place is better anyway